ionoscloud/resource_server.go

```diff
@@ -311,6 +311,12 @@ func resourceServer() *schema.Resource {
 					},
 				},
 			},
+			"vm_state": {
+				Type:             schema.TypeString,
+				Optional:         true,
+				Description:      "The power states of the Server: RUNNING, SHUTOFF, SUSPENDED (cube)",
```

Comment:
Maybe change this "The power states of the Server: RUNNING, SHUTOFF, SUSPENDED (cube)" ->  "The power states of the Server: RUNNING, SHUTOFF, SUSPENDED. SUSPENDED state is only valid for cube"